BP/common/culture/cultures/00_central_germanic.txt

```diff
@@ -0,0 +1,1772 @@
+			Haldemar Hartmut Hartwig Harduin Hildebald Hildebold Hildegaud Hildevold Hildoin Hucbert Hugbert
+			Imnachar Ingalbert Ingomer Ivo
+			Karl Karloman
+			Lambert Lantbert Leudast Lothar
+			Magnachar Magneric Mainard Mallobaudes Marachar Marcomer Marell Martin Maurifi Meginhard Merogais Merovech Munderic
+			Niebelung
+			Odelric Odolric Otbert Otgeri Otker
+			Pepin Pharamond Pippin
+			Radulf Ragambald Ragena Ragenard Raginari Ragnachar Ragnald Ragno Raimbold Rainald Ramnulf Rathar Raynold Reginari Ricchar Rignomer Roland Robert Rotbert
+			Segoin Seguin Sicbald Sichar Sicland Sicleard Siclevold Sigebald Sigebert Sigeric Sigismund Sigobert Sinop Sunnegisil Sunno
+			Tancrad Tancred Tassilo Teotbert Tetbert Teutbald Teutbert Theoderic Theoric Theudebald Theudemeres Theuderic Theudoald Theutbald Trutgaud
+			Vuitard Vulfari Vulframn Vulvari
+			Waltgaud Werinbert Wilbert Willichar Wolfari
+		}
+		female_names = {
+			Adalgardis Adallinda Adaltrude Adaluildis Adelaidis Airsenda Albofleda Albrada Alda Aldegonde Aliberta Alitrudis Ansegudis Ansegundis Anstrude Arsindis Audofleda Audovera Austreberta Austrechild
+			Balthild Begga Beretrude Bernegildis Bertenildis Berthefled Berthefried Berthegund Berthildis Bertilla Bertrada Bladovildis Brunhild Burgundofara
+			Celsa Celsovildis Cesaria Chlodosind Chlothsinda Clotild Creada
+			Dagena
+			Eldesendis Ermengardis Ermengildis Ermensindis Eustadiola
+			Faileuba Faregildis Fastrada Framberta Fredegunde Frolaica Frotberga Frotlildis Frotlina
+			Galswinth Gaudildis Gautlindis Genovefa Gersvinda Gertrude Gisela Glodesind Goiswinth Gotberga Gundrada
+			Halderudis Harildis Hildegarde Hildegardis Hildesendis Hiltrude
+			Illegardis Ingitrude Ingohildis Ingunde Itta
+			Landina Lanthechilde Lantsida Leubast Leubovera Leutberga Leutgardis Liutgarde
+			Madelgarde Magnatrude Marcatrude Marcovefa Martinga Monegund Morberga
+			Radegund Rictrude Rigunth Rosamund Rothaide Rotrude Ruothilde Rusticula
+			Sadalberga Siclehildis Sigalsis
+			Theodelinda Theoderada
+			Ultrogotha
+			Vuldretrada
+			Waltrude
+		}
+		dynasty_of_location_prefix = "dynnp_von"
+
+		# Chance of male children being named after their paternal or maternal grandfather, or their father. Sum must not exceed 100.
+		pat_grf_name_chance = 50
+		mat_grf_name_chance = 5
+		father_name_chance = 10
+		patronym_suffix_male = "dynnpat_suf_son"
+		patronym_suffix_female = "dynnpat_suf_son"
+		
+		# Chance of female children being named after their paternal or maternal grandmother, or their mother. Sum must not exceed 100.
+		pat_grm_name_chance = 10
+		mat_grm_name_chance = 50
+		mother_name_chance = 5
+		
+		ethnicities = {
+			25 = caucasian_blond
+			15 = caucasian_ginger
+			35 = caucasian_brown_hair
+			25 = caucasian_dark_hair
+		}
+
+		mercenary_names = {
+			# { name = "mercenary_company_great_company" coat_of_arms = "mc_great_company" } # Great Company No Exist
```

Comment:
What is this comment referring to?

Reply:
The Great company was a historic company that well doesn't exist at start so like I left it out.

Reply:
Ok, it can be removed entirely actually.

Reply:
Okay then

Reply:
Could you remove this line?

Reply:
Done